File: multiplatform-benchmark-java-graal/src/main/java/BenchmarkMath.java
public class BenchmarkMath {

    void benchMath(int iterations) {
        var c = 0.0;
        for (int i =0; i < iterations; i++) {
            c += (i / 2.0);
            c += (i % 3.0);
            c += Math.sqrt(i * 3.0);
            c += 1;
        }
    }
}

File: multiplatform-benchmark-kotlin/src/commonMain/kotlin/BenchmarkMath.kt
import kotlin.math.acos
import kotlin.math.sqrt

class BenchmarkMath {

    fun benchMath(iterations: Int): Double {
        var c = 0.0
        for (i in 0..iterations) {
            c += (i / 2.0)
            c += (i % 3.0)
            c += sqrt(i * 3.0)
            c += 1
        }
        return c
    }

}